lib/checkIssueProject.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @param {import('probot').Context} context
+ */
+const checkAssignedProject = async(context) => {
+    // TODO - check if issue has a project assigned or not
+    const issueComment = context.issue({ body: 'Issue should be added to a project' })
```

Comment:
I believe the comment body should ping a specific user?
What do you think @vojtechjelinek?

Reply:
Agreed.